docs/sphinx_doc/source/data_juicer.ops.filter.rst

```diff
@@ -121,6 +121,14 @@ data\_juicer.ops.filter.image\_text\_similarity\_filter
    :undoc-members:
    :show-inheritance:
 
+data\_juicer.ops.filter.image\_watermark\_filter
```

Comment:
Need to add video_watermark_filter in this doc as well.

Reply:
Done